Integrate-Kobiton.md

```diff
@@ -0,0 +1,156 @@
+# Run Kobiton Automation Test on CircleCI
+
+This document will guide you how to use CircleCI to execute automation test on Kobiton devices everytime you push a commit to GitHub.
+
+## Table of contents
+- [A. Preparation](#a-preparation)
+    - [1. Getting Kobiton Username and API key](#1-getting-kobiton-username-and-api-key)
+    - [2. Samples](#2-samples)
+- [B. Setup](#b-setup)
+    - [Purposes](#purposes)
+    - [1. Setting Kobiton Username and API Key](#1-setting-kobiton-username-and-api-key)
+    - [2. Getting Kobiton device desired capabilities](#2-getting-kobiton-device-desired-capabilities)
+    - [3. Configuring automation test script](#3-configuring-automation-test-script)
+- [C. Automation Test Execution](#c-automation-test-execution)
+    - [1. Executing the automation test](#1-executing-the-automation-test)
+    - [2. Fetching test session data using Kobiton REST API](#2-fetching-test-session-data-using-kobiton-rest-api)
+- [D. Feedback](#d-feedback)
+
+## A. Preparation
+### 1. Getting Kobiton Username and API key
+Kobiton Username and API key are required for authenticating with Kobiton REST API.
+
+> If you don't have a Kobiton account, visit https://portal.kobiton.com/register to create one.
+
+To get your Kobiton Username and API Key, follow instruction at `IV. Configure Test Script for Kobiton` section on [our blog article](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/).
+
+### 2. Samples
+In this repository, we have already provided samples for executing automation test in Kobiton:
+- Script for executing automation test on Kobiton iOS devices.
+- Script for executing automation test on Kobiton Android devices.
+- CircleCI configuration file.
+
+Link to the sample: https://github.com/kobiton/CircleCI-HockeyApp-Appium-ReactNative/tree/master/sample
+
+In this guide, as we will be using the provided example, therefore, follow these steps below:
+
+1. Fork this repository https://github.com/kobiton/CircleCI-HockeyApp-Appium-ReactNative
+
+2. Synchronize the forked repository with CircleCI. If you don't know how to do it, refer to instruction at [CircleCI document](https://circleci.com/docs/2.0/getting-started/)
+
+3. Follow next sections to get desired capabilities and configure all necessary variables for running automation test.
```

Comment:
Replace this with `Clone the forked repo and <do stuff>` because the user must clone the repo in order to edit the scripts.